src/class/SimpleData.ts

```diff
@@ -603,6 +604,36 @@ export default class SimpleData {
         return this
     }
 
+    @logCall()
+    addRank({
+        rankTitle,
+        rankBy,
+        sort,
+        locale,
+        handleTies,
+        overwrite = true,
+    }: {
+        rankTitle: string
+        rankBy?: string | string[]
+        sort?: true | false
```

Comment:
"sort" should probably have two options: ascending or descending. If undefined, it will not be sorted.